bundles/org.eclipse.emf.emfstore.server/src/org/eclipse/emf/emfstore/internal/server/accesscontrol/authentication/verifiers/PasswordVerifier.java

```diff
@@ -80,7 +80,7 @@ protected String preparePassword(String password) throws ServerKeyStoreException
 	protected boolean verifySuperUser(String username, String password) {
 		final ESPasswordHashGenerator passwordHashGenerator = AccessControl.getESPasswordHashGenerator();
 		if (hash == null && salt == null) {
```

Comment:
Should this maybe be an `||` instead of an `&&`. As I understood it, both are needed